types/types.go

```diff
@@ -256,7 +256,10 @@ type Info struct {
 	Runtimes           map[string]Runtime
 	DefaultRuntime     string
 	Swarm              swarm.Info
-	LiveRestore        bool
+	// Determine whether containers should be kept running when the
```

Comment:
That's only for routine I think. From the page:
```
type T struct {
    name    string // name of the object
    value   int    // its value
}
```

But I don't mind updating it.